src/style.css

```
@@ -1 +1,29 @@
 /* add styles here */
+*{
+  margin: 0;
+  }
```

Fix code style everywhere
```suggestion
* {
  margin: 0;
}
```

---

[ось](https://github.com/mate-academy/layout_stars/blob/master/checklist.md) чекліст по тому, як має виглядати код. Важливо притримуватись цих рекомендацій - адже це не тільки покращує читабельність, а й коли в командних проєктах налаштовані певні конфігурації - дасть змогу уникати постійних помилок